app/models/reminder.rb

```diff
@@ -4,9 +4,14 @@ class Reminder < ApplicationRecord
 
   scope :email_verified, -> { where(email_verified: true) }
   scope :not_yet_sent, -> { where(email_sent_at: nil) }
+  scope :by_journey, ->(journey) { where(journey: journey.to_s) }
   scope :inside_academic_year, -> { where(itt_academic_year: AcademicYear.current.to_s) }
   scope :to_be_sent, -> { email_verified.not_yet_sent.inside_academic_year }
 
+  def journey
+    super.constantize
+  end
```

Comment:
not sure how i feel about this and feel it might cause subtle issues. much like its common to have a column `user_id` which then backs a `record#user` method with an association. maybe the column should be `journey_class` and the method `#journey` rather an overloading the single method. 

happy to be overruled on this though

but yes definitely better to store type on the Reminder to deal with the different journeys

Reply:
Good feedback, will implement your suggestions.